### ISSUE TITLE
Can't delete scenario

### ISSUE THREAD

**Issue by daren-thomas** [architecture-building-systems/CityEnergyAnalyst#2811]
**Describe the bug**
I can't delete a scenario in my project using the GUI. The GUI doesn't show any error messages, but if I start `dashboard.bat` before starting the GUI, I see the following error:

```
[2020-08-03 12:30:12,499] ERROR in app: Exception on /api/project/scenario/kleinalbis [DELETE]
Traceback (most recent call last):
  File "C:\Users\darthoma\Documents\CityEnergyAnalyst-3.7.0\Dependencies\Python\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\darthoma\Documents\CityEnergyAnalyst-3.7.0\Dependencies\Python\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\darthoma\Documents\CityEnergyAnalyst-3.7.0\Dependencies\Python\lib\site-packages\flask_restplus\api.py", line 325, in wrapper
    resp = resource(*args, **kwargs)
  File "C:\Users\darthoma\Documents\CityEnergyAnalyst-3.7.0\Dependencies\Python\lib\site-packages\flask\views.py", line 88, in view
    return self.dispatch_request(*args, **kwargs)
  File "C:\Users\darthoma\Documents\CityEnergyAnalyst-3.7.0\Dependencies\Python\lib\site-packages\flask_restplus\resource.py", line 44, in dispatch_request
    resp = meth(*args, **kwargs)
  File "C:\Users\darthoma\Documents\CityEnergyAnalyst-3.7.0\Dependencies\Python\lib\site-packages\cea\interfaces\dashboard\api\project.py", line 209, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\darthoma\Documents\CityEnergyAnalyst-3.7.0\Dependencies\Python\lib\site-packages\cea\interfaces\dashboard\api\project.py", line 245, in delete
    config.scenario_name = ''
  File "C:\Users\darthoma\Documents\CityEnergyAnalyst-3.7.0\Dependencies\Python\lib\site-packages\cea\config.py", line 69, in __setattr__
    return general_section.parameters[cid].set(value)
  File "C:\Users\darthoma\Documents\CityEnergyAnalyst-3.7.0\Dependencies\Python\lib\site-packages\cea\config.py", line 402, in set
    encoded_value = self.encode(value)
  File "C:\Users\darthoma\Documents\CityEnergyAnalyst-3.7.0\Dependencies\Python\lib\site-packages\cea\config.py", line 843, in encode
    raise ValueError('scenario-name cannot be empty')
ValueError: scenario-name cannot be empty
```

**To Reproduce**
1. Start CEA (v3.7.0)
2. Create New Project
3. Create New Scenario
4. In Project overview, delete the Scenario, wait for the timer in the modal dialog, click "Delete"

**Expected behaviour**
I expect the project to be empty as before creating the scenario. Instead, everything just remains the same.

**Screenshots**
(see traceback above, no screenshots applicable)

**Hardware (please complete the following information):**
 - OS and version: Windows 10
 - Version of CEA you are using: v3.7.0
